hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -329,10 +341,12 @@ private function up1402_removeUnusedCaseTypes() {
    * Creates/Updates default case types to be shipped
    * with this extension
    *
+   * WARNING: Resets activity types for an assignment, discarding customization
```

Comment:
aah right ... but since Mike said it is fine to override then then I have no problem with that. you can keep things as is.